packages/flutter/lib/src/material/app_bar_theme.dart

```diff
@@ -30,13 +30,19 @@ import 'theme.dart';
 class AppBarTheme extends Diagnosticable {
   /// Creates a theme that can be used for [ThemeData.AppBarTheme].
   const AppBarTheme({
+    this.actionsIconTheme,
```

Comment:
Put below `iconTheme` since it's only adding a word to that phrase.

Reply:
Done